perf/Grpc.AspNetCore.Microbenchmarks/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using Grpc.AspNetCore.Microbenchmarks;
```

Comment:
nit: even AssemblyInfo needs copyright headers. Also to automatically add EOL, I've installed a VS/VS Code extension.